lighthouse-core/audits/non-composited-animations.js

```diff
@@ -19,8 +19,11 @@ const UIStrings = {
   =1 {# animated element found}
```

Comment:
the comment says "the number of animations" but the text  is number of _elements_. 🐱 

i personally think it'd be better to report either count of noncomp animations or % of animations that were comped

Reply:
I fixed the comment, but I still think we should list the number of animated elements rather than the number of animations.  We aren't listing the animations, only associating failure reasons with an animation name.

Animations can have more than one failure reason and animations with no name can have their failure reasons combined as duplicates.  It would not always be clear where the animation count comes from, where the animated element count would be clear.

For example, this report could have an animation count of 4 if there are three unnamed animations all operating on the property `height`:
```
<div id="asdf" .... >
Unsupported CSS Property: height
Unsupported CSS Property: width              myAnimation
```

Reply:
> animations with no name can have their failure reasons combined as duplicates

ahhhhhhh

okay yeah that makes it tricky to summarize something that's intuitive.  sg.